### PR TITLE
[DNM] Update documents

### PR DESCRIPTION
**Pull request by FelixYBW**
updated documents, add guide line to fix the jar conflict issue

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,6 +90,17 @@ If you would like to build and try Gluten with **Arrow** backend, please follow
 
 [Gluten Usage](./docs/GlutenUsage.md) listed the parameters and their default value of build command for your reference
 
+## 3.5 Jar conflicts
+
+Several libraries Gluten used is newer than Spark's, including protobuf, flatbuffers, and arrow-* (Velox backend). These libraries are compiled from source and packed into Gluten.jars. Jvm should search them from Gluten.jar firstly and load them. But for some reason jvm loads the jars from spark_home/jars which causes conflict. You may use below commands to remove the jars from spark_home/jars. We are still investigating the root cause. Welcome to share if you have good solution.
+
+```
+rm -rf $SPARK_HOME/jars/protobuf-*
+rm -rf $SPARK_HOME/jars/flatbuffers-*
```
Comment (zzcclp) [inline README.md:99]: ClickHouse Backend also dosen't need to replace this jar.

Comment (FelixYBW) [reply README.md:99]: Protobuf is used by substrait. Currently Gluten's protobuf jar is newer than Spark's. CK backends should also packed the protobut jar in gluten jar, right?

Not sure where flatbuffers are used in Gluten.

Comment (zzcclp) [reply README.md:99]: yeah, CK backends also packed the protobut jar in gluten jar, but it doesn't need to replace flatbuffers-*.jar, which may need by arrow jars.

File: docs/Velox.md [hunk 2]
```diff
@@ -75,7 +75,7 @@ sudo mkdir -p /var/lib/hadoop-hdfs/
 sudo chown <sparkuser>:<sparkuser> /var/lib/hadoop-hdfs/
 ```
 
-You also need to add configuration to the *hdfs-site.xml* as below:
+You also need to add configuration to the "*HADOOP_HOME*/etc/hadoop/hdfs-site.xml" as below:
```
Comment (jinchengchenghh) [inline docs/Velox.md:78]: The configuration for hadoop2 and hadoop3 is different

Comment (FelixYBW) [reply docs/Velox.md]: Thank you. removed the path

**Comment by FelixYBW** [comment]
@zzcclp OK to merge?

**Review by zzcclp — APPROVED** [review]
LGTM

**Pull request merged**